internal/resource/configmap_test.go

```diff
@@ -243,26 +233,20 @@ CONSOLE_LOG=new`
 				instance.ObjectMeta.Name = "rabbit-tls"
 				instance.Spec.TLS.SecretName = "tls-secret"
 
-				expectedRabbitmqConf := iniString(defaultRabbitmqConf(builder.Instance.Name) + `
-ssl_options.certfile  = /etc/rabbitmq-tls/tls.crt
-ssl_options.keyfile   = /etc/rabbitmq-tls/tls.key
-listeners.ssl.default = 5671
-
-management.ssl.certfile   = /etc/rabbitmq-tls/tls.crt
-management.ssl.keyfile    = /etc/rabbitmq-tls/tls.key
-management.ssl.port       = 15671
-
-prometheus.ssl.certfile = /etc/rabbitmq-tls/tls.crt
-prometheus.ssl.keyfile  = /etc/rabbitmq-tls/tls.key
-prometheus.ssl.port     = 15691
-
-management.tcp.port     = 15672
-prometheus.tcp.port     = 15692
-
-`)
+				expectedRabbitmqConf := iniString("ssl_options.certfile  = /etc/rabbitmq-tls/tls.crt\n" +
```

Comment:
What's the benefit of switching from comparing the config as a string to using iniString to construct the config on the fly? Personally, I found the `initString()` section harder to understand since it include new line characters `\n` manually and I need to parse it in my brain to understand what we are comparing 🤔  Thoughts?

Reply:
We were using `initString()` before this PR. I'm not saying your comment is not valid, but it should be addressed in a different PR.

I'm switing to concatenating strings using `+` over a multi-line string because the multi-line string breaks folding in NeoVim :v:

Reply:
I know that we are using iniString() before. What I meant was we were not using iniString() to construct the expected configurations, but rather concatenating most of the string. I think having the expected configurations to just be the plain string as it was before this change is easier to read. I disagree that this is a separate issue. 

I'm not sure whether I would like to merge this change for an editor issue. Maybe @ansd could add his perspective here since he also uses vim?

Reply:
Just had a call with @Zerpet:
`foldmethod=indent` doesn't work with these multi line strings (because it only checks indentation).
`foldmethod=syntax` does work (since it folds by Golang syntax, e.g. collapsing an entire test case).

Reply:
Awesome! I think we can revert some of these changes now considering vim folding problem is solved 👍